docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -10,263 +16,91 @@ By working in your local environment, you can take advantage of features within
 
-If you prefer to develop a projects in notebooks rather than an in an IDE, you should follow our guide on [how to develop a Kedro project within a Databricks workspace](./databricks_notebooks_development_workflow.md) instead.
-
-``` {note}
-[Databricks now recommends](https://docs.databricks.com/en/archive/dev-tools/dbx/index.html) that you use now use Databricks asset bundles instead of dbx. This Kedro deployment documentation has not yet been updated but you may wish to consult [What are Databricks Asset Bundles?](https://docs.databricks.com/en/dev-tools/bundles/index.html) and [Migrate from dbx to bundles](https://docs.databricks.com/en/archive/dev-tools/dbx/dbx-migrate.html) for further information.
+```{note}
+If you prefer to develop projects in notebooks rather than in an IDE, you should follow our guide on [how to develop a Kedro project within a Databricks workspace](./databricks_notebooks_development_workflow.md) instead.
 ```
 
 ## What this page covers
 
 The main steps in this tutorial are as follows:
 
-- [Create a virtual environment and install and configure dbx.](#install-kedro-and-dbx-in-a-new-virtual-environment)
-- [Create a new Kedro project using the `databricks-iris` starter.](#create-a-new-kedro-project)
-- [Create a Repo on Databricks and sync your project using dbx.](#create-a-repo-on-databricks)
-- [Upload project data to a location accessible by Kedro when run on Databricks (such as DBFS).](#upload-project-data-to-dbfs)
-- [Create a Databricks notebook to run your project.](#create-a-new-databricks-notebook)
-- [Modify your project in your local environment and test the changes on Databricks in an iterative loop.](#modify-your-project-and-test-the-changes)
+- [Prerequisites](#prerequisites)
+- [Set up your project](#set-up-your-project)
+- [Create the Databricks Asset Bundles](#create-the-databricks-asset-bundles-using-kedro-databricks)
+- [Deploy Databricks Job](#deploy-databricks-job-using-databricks-asset-bundles)
+- [Run Databricks Job](#how-to-run-the-deployed-job)
 
 ## Prerequisites
 
 - An active [Databricks deployment](https://docs.databricks.com/getting-started/index.html).
 - A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
-- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine in order to create a virtual environment with a specific version of Python (>= 3.9 is required). If you have Python >= 3.9 installed, you can use other software to create a virtual environment.
+- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine to create a virtual environment with Python >= 3.9.
 
 ## Set up your project
 
 ### Note your Databricks username and host
 
-Note your Databricks **username** and **host** as you will need it for the remainder of this guide.
```

Comment:
This section is now empty?

Reply:
And a few others

![image](https://github.com/user-attachments/assets/28a3847a-804e-413a-912c-79beb478cf39)

Reply:
hm, not sure why I missed this. It's the same as the dbx docs so I intend to just reuse the same content.